cmd/argo/commands/lint.go

```diff
@@ -37,32 +38,41 @@ func NewLintCommand() *cobra.Command {
 						return err
 					}
 				}
-				fmt.Printf("%s is valid\n", file)
+				fmt.Printf("%s is valid \u2713\n ", file)
 				return nil
 			}
-
 			for _, file := range args {
 				stat, err := os.Stat(file)
 				errors.CheckError(err)
 				if stat.IsDir() {
-					err := filepath.Walk(file, func(path string, info os.FileInfo, err error) error {
+					filepath.Walk(file, func(path string, info os.FileInfo, err error) error {
 						fileExt := filepath.Ext(info.Name())
 						switch fileExt {
 						case ".yaml", ".yml", ".json":
 						default:
 							return nil
 						}
-						return lint(path)
+						err = lint(path)
+						if err != nil {
+							unvalidWf = fmt.Errorf("unvalid workflow/workflows found")
+							log.Warn(fmt.Errorf("%w \u2717", err))
+						}
+						return nil
 					})
-					errors.CheckError(err)
 				} else {
 					err := lint(file)
 					if err != nil {
-						log.Warn(err)
+						unvalidWf = fmt.Errorf("unvalidWf workflow/workflows found")
+						log.Warn(fmt.Errorf("%w \u2717", err))
 					}
 				}
+				err = nil
+			}
+			if unvalidWf != nil {
+				errors.CheckError(unvalidWf)
+			} else {
```

Comment:
This `else` is not needed, since `errors.CheckError` will stop execution with a `fatal`.